packages/core/src/api/evm/EVMSignTypedData.ts

```
@@ -203,6 +204,30 @@ export default class EVMSignTypedData extends BaseMethod<EVMSignTypedDataParams>
       }
     }
 
+    if (response.type === 'EthereumGnosisSafeTxRequest') {
+      const { data } = this.params;
+      const param = {
+        to: data.message.to,
+        value: formatAnyHex(new BigNumber(data.message.value).toString(16)),
+        data: stripHexStartZeroes(formatAnyHex(data.message.data)),
+        operation: parseInt(data.message.operation),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)